### ISSUE TITLE
Prevent teams from issuing feedback for silent rounds

### ISSUE THREAD

**Issue by philipbelesky** [JakobReiter/Tabbie2#36]
Apologies for the open ended feedback request. I also don't have a dev environment working atm where I can replicate things so I'm adding this with 100% certainty that this already isn't the case.

Anyway, from what I could tell from Dutch Worlds it seems like teams can enter feedback on their judges during silent rounds.

**Comment by JakobReiter** [comment]
Yes that is true - currently there is not such thing in Tabbie2 like a silent round. This would have to be implemented as a flag in the database `is_silent` or so. Then the feedback could be deactivated based on this flag that the Tab Team sets in the create Round

**Comment by rscoates** [comment]
I actually disagree with this - a number of teams at WUDC wanted to submit feedback on their judges during silent rounds (usually ESL/EFL teams wanting to say that the judge in their live, but Open break dead, room looked disinterested) and assumed they couldn't. In fact, it was discussed at Council that they wanted to leave it in. I'd really rather not complicate matters by taking this out.

**Comment by philipbelesky** [comment]
Ah that seems fair. Should I close this "wont fix" or perhaps there might be some later need to mark silent rounds that this could be re-appropriated for?

**Comment by rscoates** [comment]
I think having a 'silent round' tag is a good idea, especially for displaying info on the judge's interface, and I suppose you could disable feedback for these rounds, but I wouldn't have it in by default.
